cmd/poller/poller.go

```diff
@@ -65,6 +66,8 @@ var (
 	logMaxMegaBytes = logging.DefaultLogMaxMegaBytes // 10MB
 	logMaxBackups   = logging.DefaultLogMaxBackups
 	logMaxAge       = logging.DefaultLogMaxAge
+	asupInterval    = "1m" // send every 1
```

Comment:
does this send every 1 minute or every 1 month?

Reply:
Schedule testing is in-progress.

Reply:
1m would send for every minute, actual interval will be decided later.